### PR TITLE
CLIP-1767: Replica count for Synchrony; defaultMode made configurable in volumes

### PR DESCRIPTION
**Pull request by bianchi2**
This PR makes it possible to scale Synchrony sts independently from Confluence.

Also, defaultMode becomes configurable which is handy when mounting secrets and configmaps as executable files. It defaults to 484 (decimal) which is 0744 octal.

## Checklist
- [x] I have added unit tests
- [x] I have applied the change to all applicable products
- [ ] I have added the change description to the `changelog.md` and `Chart.yaml` files
- [x] (Atlassian only) I have run the E2E test (if applicable)
- [ ] (Atlassian only) Internal Bamboo CI is passing

### CODE REVIEW

File: src/main/charts/confluence/values.yaml [hunk 1]
```diff
@@ -375,6 +375,11 @@ volumes:
   # those needs to be defined in synchrony.additionalVolumeMounts
   #
   additionalSynchrony: []
+  
+  # -- Mode bits used to set permissions on created files by default. Must be an octal value between 0000 and 0777 or a decimal value between 0 and 511
+  # Typically overriden in volumes from Secrets and ConfigMaps to make mounted files executable
+  #
+  defaultMode: 484
```
Comment (errcode1202) [inline src/main/charts/confluence/values.yaml:382]: Minor - would it be clearer if this was renamed to `defaultPermissionsMode`

**Review by errcode1202 — APPROVED** [review]

File: src/main/charts/confluence/values.yaml (hunk 1 above)
Comment (bianchi2) [reply src/main/charts/confluence/values.yaml:382]: I used the name as it appears in K8S API assuming that users who are going to override it know exactly what they are doing. Renaming it should not be a big deal, I just wanted to stick to kube api names.

Comment (bianchi2) [reply src/main/charts/confluence/values.yaml]: Renamed

File: src/main/charts/confluence/values.yaml [hunk 2]
```diff
@@ -375,6 +375,11 @@ volumes:
   # those needs to be defined in synchrony.additionalVolumeMounts
   #
   additionalSynchrony: []
+  
+  # -- Mode bits used to set permissions on created files by default. Must be an octal value between 0000 and 0777 or a decimal value between 0 and 511
+  # Typically overridden in volumes from Secrets and ConfigMaps to make mounted files executable
+  #
+  defaultPermissionsMode: 484
```
Comment (nanux) [inline src/main/charts/confluence/values.yaml:382]: Might be worth leaving comment # decimal 484 == octal 0744

**Review by nanux — APPROVED** [review]

**Review by errcode1202 — APPROVED** [review]

**Pull request merged**